bin/sozo/src/args.rs

```diff
@@ -50,6 +55,24 @@ impl SozoArgs {
             Verbosity::Quiet
         }
     }
+
+    pub fn init_logging(&self) -> Result<(), Box<dyn std::error::Error>> {
+        const DEFAULT_LOG_FILTER: &str = "info,server=debug,\
+                                          blockifier=off,jsonrpsee_server=off,\
+                                          hyper=off,messaging=debug,node=error";
```

Comment:
```suggestion
        const DEFAULT_LOG_FILTER: &str = "info,hyper=off,scarb=off";
```
`scarb` and `hyper` are super verbose in trace. Having them disabled by default enhance trace reading.